gateway/requirements.txt

```diff
@@ -14,4 +14,4 @@ requests
 flask_sqlalchemy
 flask_migrate
 flask_testing
-psycopg2
```

Comment:
Could you add `psycopg2-binary` package?

Reply:
done